profiler/shakesapp/README.md

```diff
@@ -0,0 +1,32 @@
+# Profiler Shakespeare Application in Go
+
+This application starts a server which will return the number of times a word or
+phrase appears in the works of Shakespeare, and sends queries to that server to
+create load.
+
+Profiler is enabled for this application, and can be used to identify
+opportunities to optimize the server code.
+
+## Running the application
+
+This application should be run with Go 1.14. Once that version of Go is
+installed:
+
+*   Get the application code:
+
+    ```sh
+    git clone https://github.com/GoogleCloudPlatform/golang-samples.git
+    cd golang-samples/getting-started/profiler/shakesapp
```

Comment:
I am not sure why this is under getting-started. This is a how-to app, not a getting started I think. There is already https://github.com/GoogleCloudPlatform/golang-samples/tree/master/profiler/profiler_quickstart, that one is getting started. I would put it just under profiler/. I think it is also preferred that all profiler apps are in a single place, in a single nice flat list of directories to avoid future diversion of apps.

Reply:
Done.